### ISSUE TITLE
Selection function

### ISSUE THREAD

**Issue by drphilmarshall** [astrosonnen/allZeLenses#14]
Hey @astrosonnen - Tom C has a paper coming out where he and a student look at the impact of selection effects on time delay cosmological inference (he just showed some results here at the H0LiCOW/STRIDES meeting). I think we can speak to their results a bit. The key question is: if we focus primarily on quads, how do we recover from this selection bias? I wonder: do we need to add anything to our analysis to be able to comment on this question?

**Comment by astrosonnen** [comment]
That's right, so far we've been assuming that the population of massive galaxies and that of time-delay lenses are essentially the same. Or to be precise, we're assuming that the model describing the distribution of lens properties is a fair description of the population of time-delay lenses.
There are a couple of things we could do.
1- Include selection effects when making the mocks and see if we can still get an unbiased H0
2- Make an explicit distinction between intrinsic distribution and lensing-selected distribution following Sonnenfeld et al. (2015)
